pkgs/development/libraries/libnfsidmap/default.nix

```diff
@@ -2,16 +2,18 @@
 
 stdenv.mkDerivation rec {
   name = "libnfsidmap-0.25";
-  
+
   src = fetchurl {
     url = "http://www.citi.umich.edu/projects/nfsv4/linux/libnfsidmap/${name}.tar.gz";
     sha256 = "1kzgwxzh83qi97rblcm9qj80cdvnv8kml2plz0q103j0hifj8vb5";
   };
 
-  preConfigure =
-    ''
-      configureFlags="--with-pluginpath=$out/lib/libnfsidmap"
-    '';
+  configureFlags = [
+    "--with-pluginpath=$out/lib/libnfsidmap"
```

Comment:
This literally passes `--with-pluginpath=$out/lib/libnfsidmap` to configure, i.e. the variable doesn't get expanded.